.buildkite/commands/log-outdated-pods.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+
+RELEASE_NUMBER=$1
+
+if [[ -z "${RELEASE_NUMBER}" ]]; then
+    echo "Usage $0 <release number>"
+    exit 1
+fi
+
+echo '--- :git: Checkout release branch'
+.buildkite/commands/configure-git-for-release-management.sh
+.buildkite/commands/checkout-release-branch.sh "$RELEASE_NUMBER"
+
+echo '--- :ruby: Setup Ruby tools'
+install_gems
+
+echo '--- :closed_lock_with_key: Access secrets'
+bundle exec fastlane run configure_apply
```

Comment:
Does this script use any secret?

Reply:
Good point!

It shouldn't.

Testing it via https://buildkite.com/automattic/wordpress-ios/builds/20525

Reply:
It worked. 

<img width="1217" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/d704cfd6-d445-404a-9031-89dc64d95aa6">

Removed the instruction in d7f8d961ca